src/query/api/v1/handler/prometheus/native/read_test.go

```diff
@@ -36,11 +35,13 @@ import (
 	"github.com/m3db/m3/src/query/models"
 	"github.com/m3db/m3/src/query/parser"
 	"github.com/m3db/m3/src/query/storage"
+	"github.com/m3db/m3/src/query/storage/m3/storagemetadata"
 	"github.com/m3db/m3/src/query/storage/mock"
 	"github.com/m3db/m3/src/query/test"
 	"github.com/m3db/m3/src/x/instrument"
 	xtest "github.com/m3db/m3/src/x/test"
 
+	"github.com/golang/mock/gomock"
```

Comment:
Surprised linter isn't complaining about third party libs coming before internal/local imports?

Reply:
The linter is failing currently, so it probably is 😆

Reply:
Oh actually, this third party lib is after the internal/local imports. This is in the last group.